Bottom: 511e6cec3fa5d376b61f4322af5766c9e77b62a6
Top:    6b0c2eb0865e367156f75fa9adbb3c43b4518331
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:16:18 +0000

Export multi-body parts, not a bag of loose parts due to our incomplete step assy emission code.


---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 76ab121..075f811 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -225,8 +225,8 @@ step_do_export (HID_Attr_Val * options)
   } while ((piece = piece->f) != board_outline);
   poly_Free (&board_outline);
 
-//  object3d_list_export_to_step_part (board_outline_list, temp_pcb_filename);
-  object3d_list_export_to_step_assy (board_outline_list, temp_pcb_filename);
+  object3d_list_export_to_step_part (board_outline_list, temp_pcb_filename);
+//  object3d_list_export_to_step_assy (board_outline_list, temp_pcb_filename);
   g_list_free_full (board_outline_list, (GDestroyNotify)destroy_object3d);
 
   if (1) {
